bitbots_body_behavior/config/body_behavior.yaml

```diff
@@ -33,3 +33,18 @@ behavior:
     # When the ball is closer than `max_kick_distance` meters
     # it is close enough to be kicked
     max_kick_distance: 0.3
+
+    # An area in which the ball can be kicked with the right foot
+    # defined by min/max x/y values in meters which represent ball positions relative to base_footprint
+    right_kick_min_x: 0
+    right_kick_min_y: 0
+    right_kick_max_x: 0.15
+    right_kick_max_y: 0.2
+
+    # An area in which the ball can be kicked with the left foot
+    # defined by min/max x/y values in meters which represent ball positions relative to base_footprint
+    left_kick_min_x: -0.15
```

Comment:
I think you confused x and y (see [REP 103](http://www.ros.org/reps/rep-0103.html#axis-orientation))

Reply:
yes... I did.